pip/req/req_file.py

```diff
@@ -12,6 +12,7 @@
 from pip._vendor.six.moves.urllib import parse as urllib_parse
 from pip._vendor.six.moves import filterfalse
 
+import pip
```

Comment:
the import pip thing was to deal with circular imports. Its possible some of them would work, but I got tired of trying :(.